integration/benchmark/cmd/ci/main.go

```diff
@@ -34,18 +35,17 @@ type LoadCase struct {
 	duration time.Duration
 }
 
-// This struct is used for uploading data to BigQuery, changes here should
-// remain in sync with tps_results_schema.json
+// This struct is used for uploading data to BigQuery.
 type dataSlice struct {
-	GoVersion           string
-	OsVersion           string
-	GitSha              string
-	StartTime           time.Time
-	EndTime             time.Time
-	InputTps            float64
-	OutputTps           float64
-	ProStartTransaction float64
-	ProEndTransaction   float64
+	GoVersion           string    `bigquery:"goVersion"`
+	OsVersion           string    `bigquery:"osVersion"`
+	GitSha              string    `bigquery:"gitSha"`
+	StartTime           time.Time `bigquery:"startTime"`
+	EndTime             time.Time `bigquery:"endTime"`
+	InputTps            float64   `bigquery:"inputTps"`
+	OutputTps           float64   `bigquery:"outputTps"`
+	ProStartTransaction float64   `bigquery:"proStartTransaction"`
```

Comment:
These will generate some conflicts when https://github.com/onflow/flow-go/pull/3163 is merged, I will push a fix for this.